File: example_solutions/javascript/test2.js
ans=(s,e)=>{for(var r=d=m=w=0,y=1900;y<=e;){r+=w==4&d==12&y>=s
w=++w%7
if(++d>29+(2773>>m&1)-(m==1)*(1+!(y%4<1&&(y%25|y%16<1))))d=++m-m
if(m==12)m=++y-y}return r}

File: example_solutions/Perl/test2.pl
sub ans{my($d,$a)=5,0;for my$y(1900..@_[1]){for my$m(0..11){$a++if($d%7==4&$y>=@_[0]);$d+=30+(2773>>$m&1);if($m==1){$d-=2;$d++if($y%4<1&($y%25>0|$y%16<1))}}}$a}